server/models.py

```diff
@@ -0,0 +1,71 @@
+from run import db
+from passlib.hash import pbkdf2_sha256 as sha256
+
+
+class UserModel(db.Model):
+    __tablename__ = 'users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    full_name = db.Column(db.String(120), nullable=False)
```

Comment:
nitpick: I think length of full_name String can be little less. Could be around 50.